websites/website/docs/guide/plugins-customization-plugin.md

```
@@ -0,0 +1,93 @@
+
+| Type             | Description                                             | Package name pattern                        |
+| ---------------- | ------------------------------------------------------- | ------------------------------------------- |
+| `generic`        | General-purpose plugin (`IPlugin`)                      | `@tsparticles/plugin-{name}`                |
+| `emitters-shape` | Custom emitter shape for `@tsparticles/plugin-emitters` | `@tsparticles/plugin-emitters-shape-{name}` |
+| `easing`         | Custom easing function                                  | `@tsparticles/plugin-easing-{name}`         |
+| `export`         | Custom export format                                    | `@tsparticles/plugin-export-{name}`         |
+| `color-manager`  | Custom color format parser                              | `@tsparticles/plugin-{name}-color`          |
+
+### Generated files
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)